### PR TITLE
update logging section #3050

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #3050 by adding new links and a new paragraph with the events IDs.

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md [hunk 1]
```diff
@@ -50,12 +50,28 @@ Windows Defender SmartScreen helps to provide an early warning system against we
 ## Viewing Windows Defender SmartScreen anti-phishing events
 When Windows Defender SmartScreen warns or blocks an employee from a website, it's logged as [Event 1035 - Anti-Phishing](https://technet.microsoft.com/scriptcenter/dd565657(v=msdn.10).aspx).
 
+
+## Viewing Windows event logs for SmartScreen
+SmartScreen also have it's own events Ids on the "Microsoft-Windows-SmartScreen/Debug"
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md]:
```suggestion
SmartScreen also have it's own events Ids on the "Microsoft-Windows-SmartScreen/Debug"
```
```suggestion
SmartScreen has it's own events IDs on the "Microsoft-Windows-SmartScreen/Debug"
```

File: windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md [hunk 2]
```diff
@@ -50,12 +50,28 @@ Windows Defender SmartScreen helps to provide an early warning system against we
...
+
+
+|EventID | Description |
+| :---:         |     :---:      |
+|1000 | Application SmartScreen Event|
+|1001 | Uri SmartScreen Event|
+|1002 | User Decision SmartScreen Event|
+
+>[!IMPORTANT]
+> This log is not enabled by default: Microsoft-Windows-SmartScreen/Debug, an required an MDM Policy to enable it.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md]:
```suggestion
> Microsoft-Windows-SmartScreen/Debug is not enabled by default. This log requires an MDM Policy to enable it.
```

File: windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md [hunk 3]
```diff
@@ -50,12 +50,28 @@ Windows Defender SmartScreen helps to provide an early warning system against we
...
+
 ## Related topics
 - [SmartScreen Frequently Asked Questions (FAQ)](https://feedback.smartscreen.microsoft.com/smartscreenfaq.aspx)
 
 - [How to recognize phishing email messages, links, or phone calls](https://www.microsoft.com/en-us/safety/online-privacy/phishing-symptoms.aspx)
 
 - [Threat protection](../index.md)
 
+- [Available Windows Defender SmartScreen Group Policy and mobile device management (MDM) settings](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-available-settings)
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md]:
```suggestion
- [Available Windows Defender SmartScreen Group Policy and mobile device management (MDM) settings](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-available-settings)
```

**Comment by mypil** [comment]
@j0rt3g4 - please merge the commit on the copy/edit review made by nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by j0rt3g4** [comment]
@mypil Done. Thank you @nenonix  :)

File: windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md (hunk 2 above)
Comment (mypil) [reply windows/security/threat-protection/windows-defender-smartscreen/windows-defender-smartscreen-overview.md]: @j0rt3g4 - have you also merged the commit for this suggested change?

**Comment by j0rt3g4** [comment]
Done

**Comment by j0rt3g4** [comment]
@Justinha  I've applied the changes I think is on your end to remove the hold and merge the content

**Comment by j0rt3g4** [comment]
I've done the requested changes but Idk how to remove the "changes requested" on there. @Justinha I'll leave that to you

**Comment by illfated** [comment]
@j0rt3g4 : from a practical point of view, the "Changes requested" section remains unchanged until the reviewer revisits the review section and changes the requested change to an approval. (This is currently the default behaviour for Github pull requests.)

**Pull request merged**